projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java

```diff
@@ -706,6 +709,14 @@ public String initContainer(@Nullable String containerName, @Nullable String con
     if (!containerDir.toFile().mkdirs()) {
       throw new BatfishException("failed to create directory '" + containerDir + "'");
     }
+    Path testrigsDir = containerDir.resolve(BfConsts.RELPATH_TESTRIGS_DIR);
+    if (!testrigsDir.toFile().mkdir()) {
+      throw new BatfishException("failed to create directory '" + testrigsDir + "'");
+    }
+    Path analysesDir = containerDir.resolve(BfConsts.RELPATH_ANALYSES_DIR);
+    if (!analysesDir.toFile().mkdir()) {
+      throw new BatfishException("failed to create directory '" + analysesDir + "'");
+    }
     return containerName;
```

Comment:
What is the motivation for proactively creating these directories? Why not just let the first call to initialize a testrig or analysis take care of it?